csuntechlab/iSTART
# STR-214 -- FE: Update thumbnail image sizing and introduction slide.

# Description
The thumbnails images on the dash should take up the full width. Also the background image on the introduction slide is improved especially for tablet. 
  
# Testing
1. Go the dashboard.
2. Resize browser window and pay attention to the thumbnail images. They should take up half the main container.
3. Select any one of the module to go to an introduction slide. Resize the browser window.
